src/s/entry

```diff
@@ -0,0 +1,83 @@
+; *
+; * RISC OS uCLib (Micro C Lib)
+; * 
+; * This  is the  entry header  file  for the uSCLib, for more details
+; * please read the documentation in  the doc directory and the readme
+; * files provided with the source.
+; *
+; * Copyright 2021 by Paolo Fabio Zaino
+; *
+; * Redistribution and use in source and binary forms, with or without
+; * modification, are permitted provided that the following conditions
+; * are met:
+; * Redistributions  of  source code  must  retain the above copyright
+; * notice, this list of conditions,  the distribution license and the
+; * following disclaimer.
+; * Redistributions  in  binary  form   must   reproduce   the   above
+; * copyright  notice,  this  list  of  conditions  and  the following
+; * disclaimer  in  the documentation  and/or other materials provided
+; * with the distribution.
+; * Neither  the  name of the author nor the names of the contributors
+; * may  be  used  to  endorse  or  promote products derived from this
+; * software without specific prior written permission.
+; *
+; * Licensed  under  the  Apache License, Version 2.0 (the "License");
+; * you  may not  use  this file except in compliance with the License
+; * the restrictions above and the disclaimer below.
+; * You may obtain a copy of the License at
+; *
+; *      http://www.apache.org/licenses/LICENSE-2.0
+; *
+; * THIS  SOFTWARE  IS  PROVIDED   BY   THE   COPYRIGHT   HOLDERS  AND
+; * CONTRIBUTORS  "AS IS"  AND  ANY  EXPRESS  OR  IMPLIED  WARRANTIES,
+; * INCLUDING,  BUT  NOT  LIMITED  TO,   THE  IMPLIED   WARRANTIES  OF
+; * MERCHANTABILITY  AND  FITNESS  FOR   A   PARTICULAR   PURPOSE  ARE
+; * DISCLAIMED.
+; * IN  NO  EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+; * FOR  ANY  DIRECT,  INDIRECT,  INCIDENTAL,  SPECIAL,  EXEMPLARY, OR
+; * CONSEQUENTIAL  DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+; * OF  SUBSTITUTE  GOODS  OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+; * OR  BUSINESS  INTERRUPTION)  HOWEVER  CAUSED  AND ON ANY THEORY OF
+; * LIABILITY,  WHETHER  IN  CONTRACT,  STRICT  LIABILITY,   OR   TORT
+; * (INCLUDING NEGLIGENCE OR OTHERWISE)  ARISING IN ANY WAY OUT OF THE
+; * USE OF THIS SOFTWARE,  EVEN  IF ADVISED OF THE POSSIBILITY OF SUCH
+; * DAMAGE.
+; *
+
+        GET     ListOpts
+        GET     Macros
+        GET     System
+
+        IMPORT  main
+        IMPORT  __RelocCode
+
+        EXPORT  __main
+        EXPORT  __rt_stkovf_split_small
+;        EXPORT  malloc
+;        EXPORT  free
+
+        AREA |Util$$Code|, CODE, READONLY, PIC
+        ENTRY
+__main
+        STMFD  sp!,{a1-a3,fp,lr}
+        MOV    r11,#0
+;        STR    a3,[lr, #malloc_ptr - continue]
+;continue
+        BL     __RelocCode
+        LDMFD  sp!, {a1-a3,fp}
+        BL     main
+        LDR    PC,[r13],#4
+
+;malloc_ptr
+;        DCD    0 ; I think this is required to track malloc next addr
```

Comment:
I'd put this e.g. in the first word of the R12-passed block; let's at least _pretend_ the code is read-only!

Reply:
k, will have a look at that.